src/main/java/service/operation/PurchaseOperation.java

```diff
@@ -0,0 +1,18 @@
+package service.operation;
+
+import data.db.Storage;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class PurchaseOperation implements OperationHandler {
+
+    @Override
+    public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction) {
```

Comment:
The parameter 'operation' is never used in the method. If it's not necessary, consider removing it to keep the method signature clean and clear.